Squablr/Views/CardView.m

```diff
@@ -30,15 +30,31 @@ - (instancetype)initWithBounds:(CGRect)frame profile:(Profile *)profile {
     if (self) {
         [self setup];
         self.profile = profile;
+        self.profilePictureIndex = 0;
+        // Creating invisible half views for the tap gesture recognizer
+        UIView *leftView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 175, 500)];
+        UIView *rightView = [[UIView alloc] initWithFrame:CGRectMake(175, 0, 175, 500)];
+        // Adding tap gesture recognizers for user photos
+        UITapGestureRecognizer *tappedOnLeftHalf = [[UITapGestureRecognizer alloc]
```

Comment:
This init is getting pretty big, I would recommend splitting this up into multiple functions. Like setupSubviews, setupGestureRecognizers, setupStyles, etc.

Reply:
Got it